python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S6903.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Using timezone aware \"datetime\"s should be preferred over using \"datetime.datetime.utcnow\" and \"datetime.datetime.utcfromtimestamp\"",
```

Comment:
There was an unaddressed comment on the RSPEC. I made some minor changes. You should probably regenerate the metadata to reflect that.